packages/react/src/themes/teams/components/HierarchicalTree/hierarchicalTreeTitleStyles.ts

```diff
@@ -8,7 +8,12 @@ const hierarchicalTreeTitleStyles = {
     cursor: 'pointer',
     color: variables.defaultColor,
     position: 'relative',
-    ...getBorderFocusStyles({ siteVariables }),
+    ':focus': {
+      outline: 0,
+    },
+    ':focus-visible': {
+      ...getBorderFocusStyles({ siteVariables })[':focus-visible'],
+    },
```

Comment:
Previous code does the same as I see

Reply:
before these changes it was not working, we are missing something.

before the removal of isFromKeyboard we had the issue that borderfocus style was applied every time. now, if you check on the stardust docsite, it won't be applied in any case (mouse or keyboard).

these changes fixes it.